<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestaurantClosingStock</class>
 <widget class="QWidget" name="RestaurantClosingStock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
   <item>
    <layout class="QVBoxLayout" name="AppbarLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="DateWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="fromDateLabel">
          <property name="text">
           <string>From Date:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="fromDateDateEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="displayFormat">
           <string>dd-MM-yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="toDateLabel">
          <property name="text">
           <string>To Date:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="toDateDateEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="displayFormat">
           <string>dd-MM-yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Group</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="groupLay"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Godown</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="godownLayout"/>
        </item>
        <item>
         <widget class="QCheckBox" name="ShowGodownTransferCheckBox">
          <property name="text">
           <string>Show Godown Transfers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="show0CheckBox">
          <property name="text">
           <string>Show 0 stock</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showTransCheckBox">
          <property name="text">
           <string>Show Transactions within Period</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="OKButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="searchLine">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>28</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="placeholderText">
        <string>Search...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="tableLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
